portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,38 @@
+//gets result based on quiz results
+//this needs to be hooked up to the algorithm and spotify api to return a result
+function getResult()
```

Comment:
Minor style: include the opening brace on the same line.
eg: `function getResult() {`

Same for other functions below